README.md

```diff
@@ -1,45 +1,92 @@
+> ⚠️ **Heavy WIP.** beware of the Dragons 🐉..
 
-> **This repository is incubating the Interplanetary Testground. 🐣**
+> **This repository is incubating the InterPlanetary Testground. 🐣**
 
-You may have noticed a few test efforts with similar names underway! Testing at
-scale is a hard problem. We are indeed exploring and experimenting a lot, until
-we land on an end-to-end solution that works for us. 
+## Description
+
+You may have noticed a few test efforts with similar names underway! Testing at scale is a hard problem. We are indeed exploring and experimenting a lot, until we land on an end-to-end solution that works for us.
 
 * Interplanetary Testbed (IPTB): https://github.com/ipfs/iptb
   * a simple utility to manage local clusters/aggregates of IPFS instances.
 * libp2p testlab: https://github.com/libp2p/testlab
   * a Nomad deployer for libp2p nodes with a DSL for test scenarios.
 
-The Interplanetary Test Ground aims to leverage the learnings and tooling
-resulting from those efforts to provide a scalable runtime environment for the
-execution of various types of tests and benchmarks, written in different
-languages, by different teams, targeting a specific commit of IPFS and/or
-libp2p, and quantifying its characteristics in terms of performance, resource
-and network utilisation, stability, interoperability, etc., when compared to
-other commits.
+The Interplanetary Test Ground aims to leverage the learnings and tooling resulting from those efforts to provide a scalable runtime environment for the execution of various types of tests and benchmarks, written in different languages, by different teams, targeting a specific commit of IPFS and/or libp2p, and quantifying its characteristics in terms of performance, resource and network utilisation, stability, interoperability, etc., when compared to other commits.
 
-The Interplanetary Test Ground aims to be tightly integrated with the software
-engineering practices and tooling the IPFS and libp2p teams rely on.
+The Interplanetary Test Ground aims to be tightly integrated with the software engineering practices and tooling the IPFS and libp2p teams rely on.
 
-## System architecture
+## Architecture
 
 Refer to the [specification](docs/SPEC.md) document.
 
+## Repo Structure
+
+```
+.
+├── README.md                       # This file
+├── main.go                         # TestGround entrypoint file
+├── cmd                             # TestGround CLI comamnds
+│   ├── all.go
+│   ├── build.go
+│   ├── daemon.go
+│   ├── enum_flag.go
+│   ├── helpers.go
+│   ├── list.go
+│   ├── run.go
+│   └── sidecar.go
+├── manifests                       # Manifests for each test Plan. These exist independent from plans to enable plans to live elsewhere
+│   ├── dht.toml
+│   └── smlbench.toml
+├── plans                           # The Test Plan. Includes Image to be run, Assertions and more
+│   ├── dht
+│   └── smlbench
+├── sdk                             # SDK available to each test plan
```

Comment:
@raulk small nitpick. Can we rename to `sdk-go` given that we will need a `sdk-js` soon as well? (Which is a Work Package of itself)